### PR TITLE
827157-Prepare UG documentation for the range area support in .NET  MAUI

### PR DESCRIPTION
**Pull request by DhanarajRajendran**
### Feature description ###

Added the User Guide for Range Area series Chart

### Output ###

Screenshot 2023-05-02 105019 (3)

## ##

Screenshot 2023-04-26 170041 (2)

### CODE REVIEW

File: MAUI/Cartesian-Charts/RangeArea.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
+---
+layout: post
+title: Range Area Chart in .NET MAUI Chart control | Syncfusion
```
Comment (florenceliliana) [inline MAUI/Cartesian-Charts/RangeArea.md:3]: title: Range Area Chart in .NET MAUI Chart Control | Syncfusion

File: MAUI/Cartesian-Charts/RangeArea.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+# Range Area Chart in .NET MAUI Chart
+
+## Range Area Chart
+
+Range Area Chart is a type of data visualization useful for displaying the relationship between two variables over time. In this Series, the area between two lines is filled to indicate a range of values, such as high and low price range or an upper and lower limit.
+
+By displaying ranges of data, range area series can make it easier to compare multiple datasets at once.
+
+To render a range area chart, create an instance of [RangeAreaSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
```
Comment (florenceliliana) [inline MAUI/Cartesian-Charts/RangeArea.md:18]: To render a range area chart, create an instance of the [RangeAreaSeries](), and add it to the [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) collection property of [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).

File: MAUI/Cartesian-Charts/RangeArea.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+
+Since the [RangeAreaSeries]() requires two Y values for each point, your data should contain both the high and low values. These high and low values specify the maximum and minimum ranges of the point.
+
+N> The Cartesian chart has [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) as its default content.
```
Comment (florenceliliana) [inline MAUI/Cartesian-Charts/RangeArea.md:22]: N> The Cartesian chart has a [Series](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html#Syncfusion_Maui_Charts_SfCartesianChart_Series) as its default content.

File: MAUI/Cartesian-Charts/RangeArea.md [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Range Area chart type in MAUI Chart](Chart-types_images/maui_range_area.png)
+
+## Enable Marker
+
+A marker, also known as a symbol, is used to determine or highlight the position of the data point. To enable markers in the series, set the [ShowMarkers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.AreaSeries.html#Syncfusion_Maui_Charts_AreaSeries_ShowMarkers) property to true.
```
Comment (florenceliliana) [inline MAUI/Cartesian-Charts/RangeArea.md:74]: A marker, also known as a symbol, is used to determine or highlight the position of the data point. To enable markers in the series, set the [ShowMarkers](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.AreaSeries.html#Syncfusion_Maui_Charts_AreaSeries_ShowMarkers) property to true.

File: MAUI/Cartesian-Charts/RangeArea.md [hunk 5]
```diff
@@ -0,0 +1,176 @@
...
+    chart.Series.Add(series);
+    this.Content= chart;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+### Marker customization
+
+In order to change the series markers appearance, create an instance of the [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.AreaSeries.html#Syncfusion_Maui_Charts_AreaSeries_MarkerSettings) property. The following properties are used to customize marker appearance.
```
Comment (florenceliliana) [inline MAUI/Cartesian-Charts/RangeArea.md:114]: In order to change the series marker's appearance, create an instance of the [MarkerSettings](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.AreaSeries.html#Syncfusion_Maui_Charts_AreaSeries_MarkerSettings) property. The following properties are used to customize the marker's appearance.

**Review by florenceliliana — CHANGES_REQUESTED** [review]

**Review by Saravanan-Madhesh — APPROVED** [review]
The changes are fine

**Pull request merged**